src/Manager.js

```diff
@@ -48,32 +52,61 @@ class GiveawaysManager extends EventEmitter {
          * The manager options
          * @type {GiveawaysManagerOptions}
          */
-        this.options = merge(defaultManagerOptions, options);
+        this.options = merge(GiveawaysManagerOptions, options || {});
+
         if (init) this._init();
     }
 
     /**
      * Generate an embed displayed when a giveaway is running (with the remaining time)
      * @param {Giveaway} giveaway The giveaway the embed needs to be generated for
-     * @param {boolean} lastChanceEnabled Whether or not to include the last chance text
+     * @param {boolean} [lastChanceEnabled=false] Whether or not to include the last chance text
      * @returns {Discord.MessageEmbed} The generated embed
      */
-    generateMainEmbed(giveaway, lastChanceEnabled) {
+    generateMainEmbed(giveaway, lastChanceEnabled = false) {
         const embed = new Discord.MessageEmbed();
         embed
-            .setAuthor(giveaway.prize)
-            .setColor(lastChanceEnabled ? giveaway.lastChance.embedColor : giveaway.embedColor)
-            .setFooter(`${giveaway.winnerCount} ${giveaway.messages.winners} • ${giveaway.messages.embedFooter}`)
+            .setTitle(giveaway.prize)
+            .setColor(
+                giveaway.pauseOptions.isPaused && giveaway.pauseOptions.embedColor
+                    ? giveaway.pauseOptions.embedColor
+                    : lastChanceEnabled
+                        ? giveaway.lastChance.embedColor
+                        : giveaway.embedColor
+            )
+            .setFooter(
+                typeof giveaway.messages.embedFooter === 'object'
+                    ? giveaway.messages.embedFooter.text?.length > 0
+                        ? giveaway.messages.embedFooter.text
+                        : ''
+                    : giveaway.messages.embedFooter,
+                giveaway.messages.embedFooter.iconURL
+            )
             .setDescription(
-                (lastChanceEnabled ? giveaway.lastChance.content + '\n\n' : '') +
-                giveaway.messages.inviteToParticipate +
-                    '\n' +
-                    giveaway.remainingTimeText +
-                    '\n' +
-                    (giveaway.hostedBy ? giveaway.messages.hostedBy.replace('{user}', giveaway.hostedBy) : '')
+                giveaway.isDrop
+                ? giveaway.messages.dropMessage
```

Comment:
regarding that. if a user sets a duration, would it maybe be better to also display it?
= no duration => not in embed
= duration => in embed too

= switch behaviour
?

Reply:
Or on the other side, could it be it gets not displayed, because there is the timestamp?
but some people maybe oversee that, hmm, idk?

Reply:
for me, a drop giveaway does not have any duration. Otherwise, users can still do it with the event when someone enters a giveaway and decide to end it if there is enough participations